config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  root 'root#index'
+  namespace :api do
+    namespace :v1 do
+      resources :greetings
+    end
+  end
+end
```

Comment:
As a requirement, let's ensure when the app is reloaded, it renders greetings randomly😉. Currently, it doesn't render the greetings in the database when on the homepage, and when on the greetings page it renders a routing error. please do well to fix it😉.

<img width="1316" alt="image" src="https://github.com/mumbereh/Rails-webpack/assets/64643995/17d39d0c-0c46-49f3-a1a8-6c12f1a80e54">